src/databricks/labs/ucx/hive_metastore/grants.py

```diff
@@ -205,9 +205,9 @@ def __init__(self, tc: TablesCrawler, udf: UdfsCrawler, include_databases: list[
         self._udf = udf
         self._include_databases = include_databases
 
-    def snapshot(self) -> Iterable[Grant]:
+    def snapshot(self, *, force_refresh: bool = False) -> Iterable[Grant]:
         try:
-            return super().snapshot()
+            return super().snapshot(force_refresh=force_refresh)
         except Exception as e:  # pylint: disable=broad-exception-caught
```

Comment:
Something for outside this PR, but do not like this broad exception

Reply:
I don't either.